lib/repl.js

```diff
@@ -1862,9 +1862,17 @@ module.exports = {
 
 ObjectDefineProperty(module.exports, 'builtinModules', {
   __proto__: null,
-  get: () => _builtinLibs,
-  set: (val) => _builtinLibs = val,
-  enumerable: true,
+  get: pendingDeprecation ? deprecate(
+    () => require(''),
```

Comment:
Thanks for your work! If pendingDeprecation is true, return`_builtinLibs` instead of require an empty string to keep the original behavior.

Reply:
```suggestion
    () => _builtinLibs,
```

Reply:
:fearful: not sure how that got there... maybe just some copy-pasta or something :sweat: 

Thanks, I've fixed it :slightly_smiling_face: